keps/sig-cluster-lifecycle/wgs/1815-componentconfig-objectmeta/kep.yaml

```diff
@@ -0,0 +1,45 @@
+title: Support ObjectMeta in ComponentConfig API's
+kep-number: 1815
+authors:
+  - "@phenixblue"
+  - "@mtaufen"
+owning-sig: sig-cluster-lifecycle
+participating-sigs:
+  - sig-cli
+status: provisional
+creation-date: 2020-05-27
+reviewers:
+  - TBD
+approvers:
+  - TBD
+see-also:
+  - NA
+replaces:
+  - NA
+
+# The target maturity stage in the current dev cycle for this KEP.
+stage: alpha
```

Comment:
It might be worth identifying this as GA, since the KEP proposes going straight to the current level of a given componentconfig API.

Reply:
@liggitt wdyt for this kind of change?

Reply:
Swapped to stable.